Mathlib/Data/BitVec/Lemmas.lean

```diff
@@ -60,14 +50,9 @@ lemma extractLsb_eq {w : ℕ} (hi lo : ℕ) (a : BitVec w) :
 
 theorem toNat_extractLsb' {i j} {x : BitVec w} :
     (extractLsb' i j x).toNat = x.toNat / 2 ^ i % (2 ^ j) := by
-  simp only [extractLsb', ofNat_eq_mod_two_pow, shiftRight_eq_div_pow]
-
-theorem getLsb_eq_testBit {i} {x : BitVec w} : getLsb x i = x.toNat.testBit i := by
```

Comment:
Where did this go?

Reply:
The RHS is now the definition of `getLsb`, so this is replaced by unfolding `getLsb`